websites/docs-smoke-test/src/@commercetools-docs/gatsby-theme-docs/overrides/markdown-components.js

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+import Card from '@commercetools-uikit/card';
+import { designSystem } from '@commercetools-docs/ui-kit';
+
+import addonComponents from './addon-components';
+
+const DummyCards = styled.div`
+  display: grid;
+  gap: ${designSystem.dimensions.spacings.m};
+  grid-auto-columns: 1fr;
+  grid-template-columns: ${(props) =>
+    `repeat(auto-fill, minmax(${props.narrow ? '200' : '250'}px, 1fr))`};
```

Comment:
```suggestion
    `repeat(auto-fill, minmax(${props.narrow ? '242' : '328'}px, 1fr))`};
```
This makes regular cards two-aside on landing pages and narrow cards three-aside in any landing page size and you can put 2,3,4,6 cards in a grid without whitespace.

It's not ideal in content but let's optimize for the landing page first. 

PS: If you think they aren't actually "Narrow" I would be fine swapping the parameter and rather having a "wide" option. 

PPS: I found two numbers that would give stable layouts in content too if the landing page had 16px more content width (248px / 378px), but that feels hacky - I would prefer to explicitly implement other sizes for cards that live in content (there should be only one card size in content pages because three-aside is too small anyways there)